modules/parks.js

```
@@ -1,553 +1,555 @@
+			});
+		}
+	});
+
+	// Update Parks API by parkId
+	app.patch(`/${ROUTE_PREPEND}/${VERSION}/parks/:parkId`, async (req, res) => {
+		const traceId = uuidv4();
+		const apiName = 'Update Parks API';
+		const { parkId } = req.params;
+		const {
+			name,
+			status = status.toUpperCase(),
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)